src/main/java/swervelib/motors/SparkMaxSwerve.java

```diff
@@ -247,7 +247,7 @@ public void configurePIDF(PIDFConfig config)
   {
 //    int pidSlot =
 //        isDriveMotor ? SparkMAX_slotIdx.Velocity.ordinal() : SparkMAX_slotIdx.Position.ordinal();
-    int pidSlot = 0;
+//    int pidSlot = 0;
```

Comment:
wut dis

Reply:
Commented out unused variable, was unsure if it has sentimental value to the author who added it :) Might remove it now...